cbrkit/sim/collections.py

```diff
@@ -190,3 +192,227 @@ def wrapped_func(query: Sequence[ValueType], case: Sequence[ValueType]) -> float
         return best_score
 
     return wrapped_func
+
+
+@dataclass
+class ListMappingSim:
+    value: float
+    local_similarities: List[float] = None
```

Comment:
The types do not match, maybe you mean one of the following?

```suggestion
    local_similarities: List[float] | None = None
```
```suggestion
    local_similarities: List[float] = field(default_factory=list)
```